### ISSUE TITLE
inquire about SOTAB results.

### ISSUE THREAD

**Issue by A11en0** [wbsg-uni-mannheim/TabAnnGPT#3]
Hi, thanks for your great work. Your contributions have been immensely helpful to me, and I find them very inspiring. I would like to inquire whether you have attempted to reproduce Doduo's performance on the SOTAB benchmark of CTA task?

**Comment by kkorini** [comment]
Hello, thank you for checking our work! 

I have added a folder containing the code to run the DODUO baseline in the folder "baselines". The hyperparameters that we use for training the DODUO model are: batch size = 32, learning rate = 5e-5, random seeds: 0, 1 and 2. The average Micro-F1 score of these three runs with the three different random seeds is reported in the paper.

I hope this helps in reproducing our results!
